src/App.js

```diff
@@ -14,11 +14,80 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    goods: ['Jam'],
+  }
+
+  goodAdd = (good) => {
+    if (!this.state.goods.includes(good)) {
+      this.setState(prevState => ({
+        goods: [...prevState.goods, good],
+      }));
+    }
+  }
+
+  goodDelete = (good) => {
+    this.setState(prevState => ({
+      goods: prevState.goods.filter(element => element !== good),
+    }));
+  }
+
+  goodClear = (good) => {
+    this.setState({ goods: [] });
+  }
+
+  render() {
+    const { goods } = this.state;
+
+    return (
+      <div className="App">
+        <h1>
+          {goods.length > 0
+            ? (`${goods.join(', ')} is selected `)
```

Comment:
![goods](https://user-images.githubusercontent.com/66570142/119457424-59fb4f80-bd44-11eb-93a6-974a950a3cd5.png)

if many goods are selected, this text should be `... ARE selected` (not `is`, because it's plural).
Think on how it may be fixed